### PR TITLE
[ITSADSSD-59297] + [ITSADSSD-59776] News cards (Landscape + Feature)

### PR DESCRIPTION
**Pull request by jptaranto**
Screenshot 2024-11-26 at 2 37 54 pm

Screenshot 2024-11-26 at 2 38 07 pm

### CODE REVIEW

**Comment by Bcrunk** [comment]
@jptaranto

1.  1 col image cards: in this version at 600px they convert into Story landscape cards. I believe the intent was for them to retain the current behaviour and convert to a standard card at mobile. [as per](https://design-system.uq.edu.au/iframe.html?args=&globals=&id=layout-card-grid--story-card-example&viewMode=story)

**Comment by Bcrunk** [comment]
Let me know if i am getting ahead of the work... I know sidebar will also impact some of this. 

I don't think any of Listing style cards should be multiple columns. They all scale down to single column. 

Screenshot 2024-11-27 at 11 44 24 AM

File: packages/card/src/scss/_story-card.scss [hunk 1]
```diff
@@ -76,49 +72,150 @@
...
+.uq-story-card--event,
+.uq-story-card--story {
+  .uq-story-card__description {
+    display: none;
+  }
+}
+
+.uq-story-card--feature {
+  .uq-story-card__title {
+    font-size: 32px;
```
Comment (Bcrunk) [inline packages/card/src/scss/_story-card.scss:143]: Core style: `core.$font-size-xxxl`

**Comment by jptaranto** [comment]
> Let me know if i am getting ahead of the work... I know sidebar will also impact some of this.
> 
> I don't think any of Listing style cards should be multiple columns. They all scale down to single column.

I don't think we should limit this via CSS - it should be a known limitation of the design system (not to use the landscape cards in the multi column card grids).

**Comment by jptaranto** [comment]
> @jptaranto
> 
>     1. 1 col image cards: in this version at 600px they convert into Story landscape cards. I believe the intent was for them to retain the current behaviour and convert to a standard card at mobile. [as per](https://design-system.uq.edu.au/iframe.html?args=&globals=&id=layout-card-grid--story-card-example&viewMode=story)

Take a look now - I believe as per figma all cards collapse to the landscape mode on mobile - except the feature card & the 1 column side by side card.

**Comment by Bcrunk** [comment]
> > @jptaranto
> > ```
> > 1. 1 col image cards: in this version at 600px they convert into Story landscape cards. I believe the intent was for them to retain the current behaviour and convert to a standard card at mobile. [as per](https://design-system.uq.edu.au/iframe.html?args=&globals=&id=layout-card-grid--story-card-example&viewMode=story)
> > ```
> 
> Take a look now - I believe as per figma all cards collapse to the landscape mode on mobile - except the feature card & the 1 column side by side card.

That looks right, Confirmed with laura that it was just the feature card & the 1 column that needed to retain style.

**Review by Bcrunk — APPROVED** [review]

**Pull request merged**